pwiz_tools/Skyline/FileUI/PeptideSearch/ImportFastaControl.cs

```diff
@@ -304,6 +306,13 @@ private static bool HasPrecursorTransitions(SrmDocument doc)
 
         public bool ImportFasta(IrtStandard irtStandard)
         {
+            if (!IsImportingResults && AutoTrain)
+            {
+                MessageDlg.Show(WizardForm,
+                    Resources.ImportFastaControl_ImportFasta_Cannot_automatically_train_mProphet_model_since_no_results_files_are_being_imported__Uncheck_this_option_in_order_to_continue_);

```

Comment:
I think you can just show a message with just an OK button when the checkbox is checked and then uncheck it immediately. Just so the user understands why it cannot be checked. They could also back up and add results files, right?

Reply:
Also, just bad form to tell a user to do something manually that you could do for them automatically. Even as implemented, it would be better to ask the user if they are sure they want to continue without the specified mProphet model than to tell them to uncheck the checkbox manually. But, I think what I am suggesting is better than that. It is like what we do when a user attempts to choose "Scheduled" in the method export form, but they lack enough retention time prediction information to achieve the scheduling.

Reply:
Okay, I wasn't really sure how to implement it but your suggestion sounds good. I fixed that behavior.